pytorch_lightning/trainer/data_loading.py

```diff
@@ -197,7 +198,9 @@ def __init__(self, num_features, dataset, *args, **kwargs):
 
     def _get_distributed_sampler(self, dataloader, shuffle):
         kwargs = self.distributed_sampler_kwargs
-        kwargs['shuffle'] = shuffle and not self.overfit_batches
+        kwargs["shuffle"] = shuffle and not self.overfit_batches
+        if "seed" not in kwargs and "PL_GLOBAL_SEED" in os.environ:
+            kwargs["seed"] = int(os.getenv("PL_GLOBAL_SEED"))
```

Comment:
```suggestion
        kwargs.setdefault("seed", int(os.getenv("PL_GLOBAL_SEED", 0)))
```

Reply:
Also, codecov is complaining about "lines #L201 - L203 were not covered by tests".

Can you double-check the tests are actually testing this?

Reply:
the test failed initially because I forgot the `int()` around the `os.getenv`. So, yes, the test touches these lines.